Plugin-Listing.md

```diff
@@ -62,4 +62,5 @@
 | `tst` | Tasks Sort by #Tag - Sort tasks on the page by hashtag | ✅ Task Automations |
 | `tt` | Tasks to Top - Bring all tasks in note to top | ✅ Task Automations |
 | `wc` | Shows word count and other numbers for the note | 🔢 Note Statistics |
+| `wd` | Insert dates of current week | 📅 Date Automations |
```

Comment:
I believe `qtn` needs to be added to `nmn.Templates/plugin.json` (which will then be reflected in this list.